.github/workflows/aws_tfhe_integer_tests.yml

```diff
@@ -25,26 +26,35 @@ on:
       request_id:
         description: 'Slab request ID'
         type: string
-      matrix_item:
-        description: 'Build matrix item'
+      fork_user:
+        description: 'User login of forked repo'
+        type: string
+      fork_git_sha:
+        description: 'Git SHA to checkout from fork'
         type: string
 
 jobs:
   integer-tests:
     concurrency:
-      group: ${{ github.workflow }}_${{ github.ref }}_${{ github.event.inputs.instance_image_id }}_${{ github.event.inputs.instance_type }}
```

Comment:
what's the reason for the github.event removal ?

Reply:
It's just a shorthand. `inputs` context is directly available, so prefixing it by `github.event` is redundant.